### PR TITLE
Added active state for button

### PR DESCRIPTION
**Pull request by helloilya**
The small addition that adds the active state for buttons. I also added a new default color to variables color list.

### CODE REVIEW

File: dist/milligram.css [hunk 1]
```diff
@@ -45,7 +45,7 @@ input[type='submit'] {
   background-color: #9b4dca;
   border: 0.1rem solid #9b4dca;
   border-radius: 0.4rem;
-  color: #ffffff;
```
Comment (vkeusebio) [inline dist/milligram.css]: @helloilya you should've kept it, @cjpatoilo wants it this way.

Comment (helloilya) [inline dist/milligram.css]: @vkeusebio and transparent color to?

**Comment by fabienwang** [comment]
Is it me or you are changing the dist/ files that are supposed to be generated and untouched?
You should modify the src/ sass files only !

**Comment by vkeusebio** [comment]
Made me to double-check, yep, you modified a dist rather than the source file.

**Comment by helloilya** [comment]
Source/src files are everything okay. I can remove last commits where I updated dist.

**Comment by cjpatoilo** [comment]
Hi @helloilya  
About this feature, can you provide a demo page somewhere as jsbin, codepen..?

**Comment by helloilya** [comment]
Hi @cjpatoilo. Demo is [here](https://jsfiddle.net/krrsn0aL/).

**Comment by fabienwang** [comment]
It's cool. Thanks @helloilya

**Comment by helloilya** [comment]
I removed commits about dist and left only with source.

File: src/_Color.sass [hunk 2]
```diff
@@ -7,3 +7,4 @@ $color-primary: #9b4dca !default
 $color-secondary: #606c76 !default
 $color-tertiary: #f4f5f6 !default
 $color-quaternary: #d1d1d1 !default
+$color-quinary: #545b61 !default
```
Comment (hedii) [inline src/_Color.sass]: it is useless to set this variable, you should use something like this in _Button.sass file:
````sass
background-color: darken($color-secondary, 20%)
border-color: darken($color-secondary, 20%)
````

20% has to be adjusted to match your color-quinary

Comment (helloilya) [inline src/_Color.sass]: I don't think so. Maybe someone will want set other color, not `$color-quaternary, 20%`.

Comment (hedii) [inline src/_Color.sass]: the active state color is almost always a variation of the hover state color.
no need to supercharge with a new variable...

Comment (helloilya) [inline src/_Color.sass]: Done.

**Comment by cjpatoilo** [comment]
@helloilya You did a great job, but I have doubt about add more feature. But I recognize the importance of this feature. BTW In my thoughts I imagine helping people to produce something lighter and very fast, thinking of all kinds of users, with minimal possible support.

For now I will close this request. Nothing prevents I reopen this request later. Anyway thank you for your PR.

**Comment by cjpatoilo** [comment]
@helloilya By default, browsers identify the state `:hover` to the state `:active`. 

I spent that time thinking about the best options to keep this request. But for now I will close this PR and it does not prevent that in the future I can reopen it again. Anyway thank you for your PR.

**Pull request closed without merging**